src/agri_ai/core/agent.py

```
@@ -102,24 +161,42 @@ async def process_message(self, user_message: str, user_id: str) -> str:
-            logger.error(f"Error processing message: {e}")
-            return f"申し訳ございません。処理中にエラーが発生しました。もう一度お試しください。\nエラー詳細: {str(e)}"
+            self.error_count += 1
+            processing_time = time.time() - start_time
+            self.total_processing_time += processing_time
+            
+            logger.error(f"Error processing message for user {user_id}: {e}")
+            
+            if isinstance(e, AgentProcessingError):
+                return str(e)
+            else:
+                return f"申し訳ございません。処理中にエラーが発生しました。もう一度お試しください。"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)